web/frontend/src/ui/LoginPage.tsx

```diff
@@ -0,0 +1,165 @@
+import {NavLink, useHistory} from "react-router-dom";
+import {Box, Button, Container, Link, TextField, Typography} from "@material-ui/core";
+import {Formik} from "formik";
+import * as Yup from "yup";
+import * as React from "react";
+import {connect} from "react-redux";
+import * as store from '../store';
+import axios from "axios";
+
+export interface LoginProps {
+    services: store.services.Services,
+    login: string;
+    password: string;
+    online: boolean;
+}
+
+function mapStateToProps(state: store.Store): LoginProps {
+    return {
+        services: state.services,
+        login: state.user.login,
+        password: state.user.password,
+        online: state.user.online,
+    };
+}
+
+const axiosInstance = axios.create({
+    baseURL: "http://127.0.0.1:5000/users/login",
+    headers: {
+        "Authorization": "Bearer dev",
+        "Content-Type": "application/json",
+    }
+})
+
+const LoginImpl: React.FC<LoginProps> = (props: LoginProps) => {
+    const history = useHistory()
+    const login = props.login
+    const password = props.password
+    const { services } = props
+    return (
+        <>
+            <Box
+                sx={{
+                    backgroundColor: 'background.default',
+                    display: 'flex',
+                    flexDirection: 'column',
+                    height: '94%',
```

Comment:
Why is height in `%`?

Reply:
Because the header height is given as a %